Gems/PhysX/Code/Editor/EditorJointConfiguration.cpp

```diff
@@ -330,6 +331,9 @@ namespace PhysX
                         "Select = Show setup display when entity is selected."
                         "Always = Always show setup display.")
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &EditorJointConfig::IsInComponentMode)
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &EditorJointConfig::m_jointname 
+                        , "Joint name"
+                        , "Usually coming from URDF")
```

Comment:
>"Usually coming from URDF"

That is very specific to be added into description of the name. It'd need to be kept generic: "Name used to easily identify the joint" or something in those lines.